pkg/development/wasm/main.go

```diff
@@ -0,0 +1,70 @@
+//go:build wasm
+// +build wasm
+
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"syscall/js"
+
+	"google.golang.org/protobuf/encoding/protojson"
+
+	"github.com/authzed/spicedb/pkg/development"
+	devinterface "github.com/authzed/spicedb/pkg/proto/developer/v1"
+)
+
+func rerr(message string) interface{} {
+	return js.ValueOf([]interface{}{nil, message})
+}
+
+func derr(derrs *development.DeveloperErrors) interface{} {
+	marshalled, err := json.Marshal(derrs)
+	if err != nil {
+		return rerr(err.Error())
+	}
+
+	return js.ValueOf([]interface{}{string(marshalled), nil})
+}
+
+func runDevelopmentOperations(this js.Value, args []js.Value) interface{} {
```

Comment:
Document the expected shape of `args` and perhaps the supported operations of the 2nd arg since this is kind of like an interface between the js and the wasm side. Or a link to the example `wasm.html` could suffice since it has an example request json.